### ISSUE TITLE
[csrng,doc] RNG compliance strategy document?

### ISSUE THREAD

**Issue by marnovandermaas** [lowRISC/opentitan#16017]
I was reading through the CSRNG documentation and was wondering where I can find the RNG compliance strategy document?
https://github.com/lowRISC/opentitan/blob/10e7b2251fc01f0bbf2934804f466310810c2f2f/hw/ip/csrng/doc/_index.md?plain=1#L10

**Comment by msfschaffner** [comment]
CC @tjaychen @moidx

**Comment by moidx** [comment]
Are you asking about RNG compliance from a certification perspective? If so, this is not something we are planning to open source as it is specific to each silicon instantiation.

**Comment by marnovandermaas** [comment]
> Are you asking about RNG compliance from a certification perspective? If so, this is not something we are planning to open source as it is specific to each silicon instantiation.

@moidx, well the public documentation refers to "our RNG compliance strategy document". I have not been able to find this anywhere. I suggest that if the document does not exist or if the document is not meant to be public that we remove it from the public facing CSRNG docs.

**Comment by msfschaffner** [comment]
I agree, if this is not publicly accessible we should probably remove the reference from the docs.

**Comment by martin-lueker** [comment]
Hi all,
   My apologies for the delay in response. This document was to be based on a presentation that I gave at the Google Mathilda campus many years ago now, and @cdgori asked me to convert it to Markdown.  It was a fairly straight forward summary of the various classes of RNG generator in AIS31 and SP-800 90A/B/C, listing their correspondences and laying out a strategy for some level of mutual compliance.  It seems however that this conversion slipped off my radar.  
  For now I'll just remove this line.

**Comment by msfschaffner** [comment]
Sounds good - although that document would be interesting to have on our page. 
Maybe we can keep this issue around so that we do not forget in the future.

**Comment by marnovandermaas** [comment]
Ok, I created a PR to remove this line from the documentation, but we can leave this issue open to remind us that we should add the mention of the RNG compliance document back in the future once it has been published.

**Comment by cdgori** [comment]
That document is/was quite handy and informative, so +1 for ultimately getting a version of it on our doc pages.

**Comment by tjaychen** [comment]
@martin-lueker just wanted to check in on this item since you probably have the most knowledge for it :)
